Xamarin.Forms.Platform.iOS/CollectionView/HorizontalTemplatedCell.cs

```diff
@@ -5,12 +5,12 @@
 
 namespace Xamarin.Forms.Platform.iOS
 {
-	internal sealed class TemplatedHorizontalListCell : TemplatedCell
+	internal sealed class HorizontalTemplatedCell : TemplatedCell
 	{
 		public static NSString ReuseId = new NSString("Xamarin.Forms.Platform.iOS.TemplatedHorizontalListCell");
```

Comment:
It might be good to rename this to match the type name - at least until we go stable.